examples/nlp/language_modeling/megatron_change_num_partitions.py

```diff
@@ -1046,10 +1281,16 @@ def main():
                 with open_dict(model.cfg):
                     model.cfg.tokenizer.model = args.tokenizer_model_path
 
-            model = cls(model.cfg, trainer).to('cpu')
+            model.cfg, restore_dict = force_cpu_model(model.cfg)
+
+            model = cls(model.cfg, trainer)
+            model = model.to('cpu')
```

Comment:
why do we need this?

Reply:
Due to NeMo bug in some layers, not all layers will be on CPU even after explicit setting of config to load on CPU - this is to force the remainder of the layers to move to cpu.

Reply:
ahh okay ://
can we go OOM because of this?

Reply:
Thankfully for GPT, only one of the embedding layers has this issue, most other layers respect the cpu_init flag. So rarely possibly to get OOM due to this on GPT. T5 yes quite possible cause there's a bunch of places that's ignoring the flag.